webpack.config.js

```diff
@@ -6,63 +6,70 @@ const CopyWebpackPlugin = require('copy-webpack-plugin');
+  console.log("DIST_BASE_URL: ", env.DIST_BASE_URL); // true
+  return {
+    context: __dirname,
+    entry: {
+      app: "./src/index.js",
     },
-    mainFiles: ['module', 'main', 'Cesium']
-  },
-  module: {
-    rules: [
-      {
-        test: /\.(js)$/,
-        exclude: /node_modules/,
-        use: ['babel-loader']
-      },
-      {
-        test: /\.css$/,
-        use: [ 'style-loader', 'css-loader' ]
+    output: {
+      filename: "app.js",
+      path: path.resolve(__dirname, "dist"),
+      // Needed to compile multiline strings in Cesium
+      sourcePrefix: "",
+    },
+    amd: {
+      // Enable webpack-friendly use of require in Cesium
+      toUrlUndefined: true,
+    },
+    resolve: {
+      alias: {
+        cesium: path.resolve(__dirname, cesiumSource),
       },
-      {
-        test: /\.(png|gif|jpg|jpeg|svg|xml|json)$/,
-        use: [ 'url-loader' ]
-      }
-    ]
-  },
-  plugins: [
-    // Allow pulling envionment variables from .env file
-    new DotenvWebpackPlugin(),
-    // Copy over index.html template into dist directory with the proper includes
-    new HtmlWebpackPlugin({
-      template: 'src/index.html'
-    }),
-    // Copy Cesium Assets, Widgets, and Workers to a static directory
-    new CopyWebpackPlugin({
-      patterns: [
-        { from: path.join(cesiumSource, cesiumWorkers), to: 'Workers' },
-        { from: path.join(cesiumSource, 'Assets'), to: 'Assets' },
-        { from: path.join(cesiumSource, 'Widgets'), to: 'Widgets' }
-      ]
-    }),
-    new webpack.DefinePlugin({
-      // Define relative base path in cesium for loading assets
-      CESIUM_BASE_URL: JSON.stringify('')
-    })
-  ],
-  mode: 'development'
-};
+      mainFiles: ["module", "main", "Cesium"],
+    },
+    module: {
+      rules: [
+        {
+          test: /\.(js)$/,
+          exclude: /node_modules/,
+          use: ["babel-loader"],
+        },
+        {
+          test: /\.css$/,
+          use: ["style-loader", "css-loader"],
+        },
+        {
+          test: /\.(png|gif|jpg|jpeg|svg|xml|json)$/,
+          use: ["url-loader"],
+        },
+      ],
+    },
+    plugins: [
+      // Allow pulling envionment variables from .env file
+      new DotenvWebpackPlugin(),
+      // Copy over index.html template into dist directory with the proper includes
+      new HtmlWebpackPlugin({
+        template: "src/index.html",
+      }),
+      // Copy Cesium Assets, Widgets, and Workers to a static directory
+      new CopyWebpackPlugin({
+        patterns: [
+          { from: path.join(cesiumSource, cesiumWorkers), to: "Workers" },
+          { from: path.join(cesiumSource, "Assets"), to: "Assets" },
+          { from: path.join(cesiumSource, "Widgets"), to: "Widgets" },
+        ],
+      }),
+      new webpack.DefinePlugin({
+        // Define relative base path in cesium for loading assets
+        CESIUM_BASE_URL: JSON.stringify(
+          env.CI ? `/tileviewer/${env.DIST_BASE_URL}/` : ""
```

Comment:
'CI' is a variable that's only true while we're running in GH actions, the 'DIST_BASE_URL' is also fed in from the deploy.yml.